Data-Structures/Array/test/MooreVotingAlgorithm.test.js

```diff
@@ -0,0 +1,11 @@
+import { MooreVotingAlgorithm } from '../MooreVotingAlgorithm'
+
+describe('Moore Voting Algorithm', () => {
```

Comment:
The case where there is no majority element should also be tested.

Reply:
added the no majority element test case